lib/typescript/httpclient/model/Message.ts

```diff
@@ -35,7 +35,7 @@ export interface Message extends Content {
   metadata?: MessageMetadata;
 }
 
-export interface SuggestedReply {
+export interface SuggestedReply extends Content {
   content: {
     text: string;
```

Comment:
Not sure if this is right. `SuggestedReply` can have any type of message inside, not just text.